bin/run

```diff
@@ -27,7 +27,6 @@ wait_for() {
 wait_for db 5432
 
 tests() {
-  mkdir -p /app/.coverage
```

Comment:
I think this'll still need to happen or at least I don't understand why the tests fail for that.